service/controller/appcatalog/v1/resource_set.go

```diff
@@ -56,7 +57,20 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 			IndexNamespace: config.IndexNamespace,
 		}
 
-		ops, err := index.New(c)
+		stateGetter, err := index.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		configOps := configmap.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+
+			Name:        config.ProjectName,
```

Comment:
The name here should be "configmap". This is what `resource.Name()` returns.

Reply:
I will make it constant

Reply:
Actually you can do that:

```suggestion
			Name:        index.Name,
```

Reply:
is it okay to have `indexv1`?

Reply:
Yes it's fine.